### PR TITLE
Add gt, gte, lt, lte predicate support to opspec

### PR DESCRIPTION
**Pull request by chrisdostert**
closes part of https://github.com/opctl/opctl/issues/984

### CODE REVIEW

File: sdks/go/opspec/interpreter/call/predicates/predicate/ge/interpret.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package ge
```
Comment (zachpuck) [inline sdks/go/opspec/interpreter/call/predicates/predicate/ge/interpret.go:1]: what is ge?

File: website/docs/reference/opspec/op-directory/op/call/predicate.md [hunk 2]
```diff
@@ -22,13 +26,37 @@ Items:
 ### exists
 A [variable-reference [string]](../variable-reference.md) defining a predicate, true when the referenced value exists.
 
+### ge
```
Comment (zachpuck) [inline website/docs/reference/opspec/op-directory/op/call/predicate.md:29]: gt?

File: sdks/go/model/opSpec.go [hunk 3]
```diff
@@ -78,6 +78,10 @@ type ParallelLoopCallSpec struct {
 type PredicateSpec struct {
 	Eq        *[]interface{} `json:"eq,omitempty"`
 	Exists    *string        `json:"exists,omitempty"`
+	LE        *[]interface{} `json:"le,omitempty"`
+	LTE       *[]interface{} `json:"lte,omitempty"`
+	GE        *[]interface{} `json:"ge,omitempty"`
```
Comment (zachpuck) [inline sdks/go/model/opSpec.go:83]: GT?

File: website/docs/reference/opspec/op-directory/op/call/predicate.md (hunk 2 above)
Comment (chrisdostert) [reply website/docs/reference/opspec/op-directory/op/call/predicate.md:29]: yep; good catch. Same for lt

**Review by zachpuck — APPROVED** [review]

**Pull request merged**